examples-bkup/chat.rs

```diff
@@ -172,3 +207,64 @@ fn print_ourinfo(crust: &mut Crust) {
         serde_json::to_string(&ourinfo).unwrap()
     );
 }
+
+fn parse_cli_args() -> CliArgs {
+    let matches = App::new("Simple chat app built on Crust")
```

Comment:
I think this should be
```suggestion
    let matches = App::new("Simple chat app built on p2p-quic")
```